### ISSUE TITLE
ContinuationTimeout is not getting picked from ConnectionFactory while creating channel.

### ISSUE THREAD

**Issue by HelioVaGator** [rabbitmq/rabbitmq-dotnet-client#821]
We are creating a `ConnectionFactory` with custom timeouts and then we are creating `Connection`

```
            var connectionFactory = new ConnectionFactory
            {
                HostName = <uri>,
                AutomaticRecoveryEnabled = true,
                NetworkRecoveryInterval = TimeSpan.FromSeconds(10),
                ContinuationTimeout = TimeSpan.FromSeconds(10),
                HandshakeContinuationTimeout = TimeSpan.FromSeconds(10),
                RequestedConnectionTimeout = TimeSpan.FromSeconds(10),
                SocketReadTimeout = TimeSpan.FromSeconds(10),
                SocketWriteTimeout = TimeSpan.FromSeconds(10),
                RequestedHeartbeat = TimeSpan.FromSeconds(10),
            };

            var connection = connectionFactory.CreateConnection();
            var publisherChannel = _connection.CreateModel();
// while inspecting `publisherChannel` has `ContinuationTimeout` of 20s
```
<img width="965" alt="connection factory" src="https://user-images.githubusercontent.com/6700486/79882236-e8015a00-840f-11ea-8252-a5e22ffbc840.png">

<img width="710" alt="channel" src="https://user-images.githubusercontent.com/6700486/79883289-5b579b80-8411-11ea-9e60-93731b47efc9.png">

Somehow https://github.com/rabbitmq/rabbitmq-dotnet-client/blob/93cdd66867c92cb281f01243d4b278c09f3c87bd/projects/client/RabbitMQ.Client/src/client/impl/Connection.cs#L1207 this line is not in effect.

**Comment by lukebakken** [comment]
I'm assuming you're using version `6.0.0` of this library. If you have time to investigate and submit a PR to fix this, it would be appreciated. If not, let me know and I will try to find time to investigate.

**Comment by HelioVaGator** [comment]
I have tried, `v4.1.3, v5.1.0 and v6.0.0` all have the same behavior. 
Sure I will definitely see if I could find the root cause.

**Comment by michaelklishin** [comment]
Addressed in #823 by @AndrzejKrol 👍

**Comment by lukebakken** [comment]
Thanks everyone!
